beginner_source/quickstart/build_model_tutorial.py

```diff
@@ -0,0 +1,153 @@
+"""
+Build Model Tutorial
```

Comment:
FYR we were initially considering simply linking to this [recipe](https://pytorch.org/tutorials/recipes/recipes/defining_a_neural_network.html) for the 'Build an NN' section. Linking here as you might find it useful to adapt from